eclipse/che-theia
# feat(port-plugins): Improve UX

### What does this PR do?
Improve UX of ports plug-in with feedback from UX team
Update icons, tooltips, labels

### Screenshot/screencast of this PR
![Screenshot 2020-11-17 at 13 47 45](https://user-images.githubusercontent.com/436777/99394305-f39dfc00-28de-11eb-9a99-dcb24a018fae.png)
![9SZGGP6Q4M](https://user-images.githubusercontent.com/436777/99282491-0a3b4900-2834-11eb-8028-753b582ff8e0.gif)


### What issues does this PR fix or reference?
Fixes https://github.com/eclipse/che/issues/18320

### How to test this PR?
Launch a workspace and look at endpoints exposed
Check tooltips, icons and labels

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### Happy Path Channel

HAPPY_PATH_CHANNEL=next

Change-Id: Ib56f8c3cf7415cd2fdcb1a39b0000de0a1bb6edb
Signed-off-by: Florent Benoit <fbenoit@redhat.com>
